src/Microsoft.ML.DnnAnalyzer/Microsoft.ML.DnnAnalyzer/DnnAnalyzer.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Transforms.TensorFlow;
+using System.Linq;
+
+namespace Microsoft.ML.DnnAnalyzer
+{
+    public static class DnnAnalyzer
+    {
+        public static void Main(string[] args)
+        {
+            using (var env = new TlcEnvironment())
+            using (var ch = env.Start("DnnAnalyzer"))
+            {
+                if (Utils.Size(args) != 1)
+                {
+                    ch.Error("Usage: dotnet DnnAnalyzer.dll <model_location>");
```

Comment:
>.dll [](start = 55, length = 4)

is dll necessary? and should it be dotnet run or just dotnet works?

Reply:
"dotnet DnnAnalyzer <model file>" didn't work.
"dotnet run DnnAnalyzer.dll <model file>" didn't work.

Is there a different syntax I should try?

---
In reply to: [218189666](https://github.com/dotnet/machinelearning/pull/862#discussion_r218189666) [](ancestors = 218189666)

Reply:
No, apparently i need to study dotnet better. dotnet run for projects only,

---
In reply to: [218535326](https://github.com/dotnet/machinelearning/pull/862#discussion_r218535326) [](ancestors = 218535326,218189666)